helm/cert-manager-app/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "certManager.name" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app.kubernetes.io/component: "controller"
+    {{- include "certManager.defaultLabels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: "controller"
+      {{- include "certManager.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: "controller"
+        {{- include "certManager.defaultLabels" . | nindent 8 }}
+      annotations:
+        prometheus.io/path: "/metrics"
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9402'
+    spec:
+      serviceAccountName: {{ template "certManager.name" . }}
+      securityContext:
+        runAsUser: {{ .Values.securityContext.userID }}
+        runAsGroup: {{ .Values.securityContext.groupID }}
+      initContainers:
+        - name: disable-webhook-validation
+          image: "{{ .Values.global.image.registry }}/{{ .Values.image.kubectl.name }}:{{ .Values.image.kubectl.tag }}"
+          imagePullPolicy: {{ .Values.image.kubectl.pullPolicy }}
+          args:
+          - label
+          - namespace
+          - {{ .Release.Namespace | quote }}
+          - certmanager.k8s.io/disable-validation=true
+          - --overwrite=true
```

Comment:
this could be a one-shot install job with a hook, but it stops people possibly shooting themselves in the foot by unlabelling the namespace in the future. additionally, the kubectl image is less than 20MB.